content/faq/fundamental/oauth.md

```diff
@@ -66,13 +98,58 @@ This redirect URI is used when a resource owner grants authorization to the clie
 - Our authorization and resource are on the different machines.
 - However, the authorization server may be the same server as the resource server or a separate entity as mentioned in https://tools.ietf.org/html/rfc6749#section-1.1
 
-# Client credentials grant type
+
+## Grant Types
+
+1. Client credentials
+
+2. Password
+
+3. Authorization code
+
+4. Authorization code with PKCE
```

Comment:
I like this. Adding a number can help a reader to understand that we have 4 grant types